pkg/line/line_test.go

```diff
@@ -403,6 +403,17 @@ func TestLineToEvents(t *testing.T) {
 		"datadog tag extension with both valid and invalid utf8 tag values": {
 			in: "foo:100|c|@0.1|#tag1:valid,tag2:\xc3\x28invalid",
 		},
+		"datadog timings with extended aggregation values": {
```

Comment:
It might be good to add tests with metrics without tags as well, invalid tests where someone submits a line with a different tagging style (e.g. SignalFX) and the dogstatsd extended aggregation, and invalid tests for extended aggregation with counters and gauges

Reply:
I believe I have now added these tests plus some extras as well as the additional logic required in the `line.go` code to pass them.

One more question. If parsing for DogStatsD tags is disabled, should a line with the extended aggregation format result in:
1. no tags but otherwise the same output as if DogStatsD parsing was enabled
2. no output as the line is considered a bad line
?

The other tests seemed to suggest that the main side effect of turning off parsing should be no tags, but since this is a DogStatsD specific feature, should this line be considered invalid altogether in such a scenario?